.github/CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 ### New features
 
 ### Improvements
-
+* Faster implementation of `hermite_multidimensional_numba` and `hermite_multidimensional_numba_grad`. [#280](https://github.com/XanaduAI/thewalrus/pull/280)
```

Comment:
This is already part of the bugfix release (and is a duplicate of line 21).
```suggestion

```

Reply:
I haven't written this part 🤔

Reply:
oh perhaps it's from when master was merged into the PR

Reply:
Oh, yes, that was probably the case.

Reply:
@ziofil Don't forget to remove this!